README.md

```diff
@@ -8,8 +8,26 @@ Version: 0.1.0
 ### Getting Started
 TODO
 
+---
 ### Endpoints
-TODO
+##### `/ecommerce/cart`
+`GET` - Get the current user's cart.
+`POST` - Add a product to the user's cart. `product_id` needs to be specified as an url parameter.
+
+##### `/ecommerce/order`
+`GET` - Get all logged users orders. If the user is not logged in, it returns nothing.
```

Comment:
`returns nothing` you mean `returns empty array` ? or what `nothing means in this context?